### DISCUSSION TITLE
Opening article when expanding sidebar group

### DISCUSSION THREAD

**Discussion by c-ehrlich** [facebook/docusaurus#9058] [Q&A]
I did some testing with actual users on our docs recently, and many of them didn't realize that the "folders" are also articles themselves. When you click the title it opens the article, but when you click the expand arrow it only expands the group. 

<img width="287" alt="CleanShot 2023-06-11 at 17 33 09@2x" src="https://github.com/facebook/docusaurus/assets/8353666/71ad8e6a-7c52-4033-b26a-505a0163ecc0">

On the other hand, for example on https://nextjs.org/docs when you click a group where the "folder" itself is also an article, it always opens that article.

Is there any way to achieve this behaviour in Docusaurus?

**Comment by slorber** [comment]
The default behavior is on purpose: we should be able to open a category without navigating it, even if the category is linking to some page. 

As a user don't you think it's preferable to keep this ability?
I'm surprised that users get confused by this considering you are the first to report this UX problem here 😅 

Now if you want the Nextjs behavior, we don't have a way to do that apart with swizzling, but that could probably be implemented easily (new category sidebar attribute + respective md frontMatter)
